### ISSUE TITLE
font size is not respecting the font size the user sets in the browser.

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#8458]
Please add a slider, e.g., in the Map Data menu, to adjust label size. Currently the only way to make
![20210416T040412](https://user-images.githubusercontent.com/1230959/114931662-1a019c80-9e69-11eb-9beb-bbfa3a06c12c.jpg)
labels bigger, CTRL++++... alas makes the window now cover fewer hectares:
![20210416T040340](https://user-images.githubusercontent.com/1230959/114931686-1ff77d80-9e69-11eb-9078-4d45b0fa0166.jpg)

In fact there can be different sliders for each data layer. And a grand slider for all. Anyway, we are talking about basic accessibility issues here.

**Comment by mxdanger** [comment]
The issue here is that the font size is not respecting the font size the user sets in the browser. This is unrelated to page zoom.

What you are proposing would be worse than adding proper text scaling support and it would go against the accessibility features already in place that users may rely on.

**Comment by matkoniecz** [comment]
adding even more settings specific to iD seems to not make much sense

we already have unfindably many settings

**Comment by tyrasd** [comment]
Yes, this could be an accessibility issue, but if map labels are too small, the rest of the text in iD's UI is probably also too small to be well readable. So, this particular issue would not solve it fully.

This might be addressable more generally by customizable map rendering stylesheets, see #11189
